tactic/pi_instances.lean

```diff
@@ -0,0 +1,53 @@
+--  -*- lean-check-visible-files: t; -*-
```

Comment:
I don't think we should add emacs comments to our files. If you want to use something use `dir-locals.el`
Also add a proper header.